scripts/python_utils.py

```diff
@@ -0,0 +1,52 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Feature detection utilities for Python 2 and Python 3."""
```

Comment:
Please add unit tests in this PR so we don't have to later.

Reply:
Hm, how to add tests for features that are only available/accessible in Python3 without actually running Python3 which we can't due the GAE dependency issues.

Reply:
Well, all these functions need to work in both `python2` and `python3`, so let's write tests that prove that.

In other words: the tests should work in python2, and after then the migration, they should continue to work without modification.

Reply:
done